compression.go

```diff
@@ -15,7 +15,7 @@ import (
 // runsqs.SQSProducer to produce the compressed message.
 // We must base64 encode the compressed message because SQS
 // supports only character data -- valid utf8 -- not raw binary data
-type CompressionSQSProducer struct {
+type CompressionSQSProducer struct { // nolint
```

Comment:
I'm pretty sure this lint check fail is for having the package name in the struct type. Could we not just rename it to `SQSProducer`

Reply:
yeah I was going to "least changes" in this PR so I didn't have to release a major version change due to broken signature.  I'm ok either way.  Opinion?

Reply:
That's completely fair. Considering this is a pretty small library and barely touched, I'm good with keeping it as is 👍